__tests__/server.test.js

```diff
@@ -68,4 +67,49 @@ describe("backend API project", () => {
       return Promise.all(methodPromises);
     });
   });
+  describe("GET /api/articles/:article_id", () => {
+    test("that a GET request to /api/articles/:article_id responds with the correct article object", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+          expect(typeof body.article).toBe("object");
+          expect(body.article).toHaveProperty("author", expect.any(String));
+          expect(body.article).toHaveProperty("title", expect.any(String));
+          expect(body.article).toHaveProperty("article_id", expect.any(Number));
+          expect(body.article).toHaveProperty("body", expect.any(String));
+          expect(body.article).toHaveProperty("topic", expect.any(String));
+          expect(body.article).toHaveProperty("created_at", expect.any(String));
+          expect(body.article).toHaveProperty("votes", expect.any(Number));
+          expect(body.article).toHaveProperty(
+            "article_img_url",
+            expect.any(String)
+          );
+        });
+    });
+    test("that path responds with 405 Method Not Allowed if method type is invalid", () => {
```

Comment:
as this is just to the endpoint /api, i wouldnt have this test grouped with these others.